bin/build_homework_function.sh

```diff
@@ -176,7 +174,7 @@ function build_homework {
     fi
 
     # Add allowed minutes in database from config if exists
-    python3 ${SUBMITTY_INSTALL_DIR}/bin/set_allowed_mins.py ${hw_build_path}/complete_config.json ${semester} ${course} ${assignment}
+    python3 "${SUBMITTY_INSTALL_DIR}/bin/set_allowed_mins.py" "${hw_build_path}/complete_config.json" "${semester}" "${course}" "${assignment}"
     set_minutes=$?
 
     if ((set_minutes != 0)); then
```

Comment:
Quote here too with dollar sign

Reply:
Is there a reason you didn't do this one?

Reply:
Are you mentioning the one on (new) line 180?  I'm by no means an expert at shell scripting, but my understanding is that line 180 compares the actual string "set_minutes" to 0, not the value of the variable `$set_minutes`.  Whether or not this line performs the task it was meant to do is a good question that I'm not entirely sure about.

Reply:
I added this code in https://github.com/Submitty/Submitty/pull/6631. The intention was that the variable set_minutes is compared with 0. This does seem to work although I definitely think it should be corrected. I must have made a typo but it worked as expected while testing. This is what I'm suggesting to change it to:
```suggestion
    if (("$set_minutes" != 0)); then
```

Reply:
Thanks for the clarification!